include/http/libcurl_http_client.h

```diff
@@ -127,6 +127,26 @@ namespace azure {  namespace storage_lite {
             check_code(curl_easy_setopt(m_curl, CURLOPT_READDATA, this));
         }
 
+        void set_input_stream_length(size_t streamlen)
```

Comment:
it is better to change the coding order to write get_is_stream_length right after set_is_stream_length for readiness.

Reply:
will do